README.md

```diff
@@ -280,6 +288,225 @@ objectives:
     that is used as the denominator. Received data is used to compare objectives
     (threshold) values to find total number of metrics.
 
+#### Alert Policy
+
+An Alert Policy allows you to define the alert conditions for a SLO.
+
+```yaml
+apiVersion: openslo/v1alpha
+kind: AlertPolicy
+metadata:
+  name: string
+  displayName: string # optional
+spec:
+  description: string # optional
+  alertWhenNoData: boolean
+  alertWhenResolved: boolean
+  alertWhenBreaching: boolean
```

Comment:
Let's go ahead and merge this for now and start iterating on follow up PRs.  Definitely open to a wording re-evaluation - I'm personally leaning towards boring old `enabled` to account for scenarios where we wouldn't even want Alerts evaluated during certain time periods.  In addition to allowing platforms to reduce the query load resulting from alert evaluation, it would also ensure that triggered alerts don't end up adding other metadata, like Annotations, to the SLO history for a disabled period of time.